### ISSUE TITLE
twisted.names.test.test_tap.OptionsTests.test_recursiveConfiguration is failing on appveyor

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8663]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#8663|
|Type|defect|
|Created|2016-07-24 02:30:38Z|
|Branch|https://github.com/twisted/twisted/tree/8663-namestaponwin|

There's a dirty reactor error:

```
<DelayedCall 0x94add00 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('d.root-servers.net', <Deferred at 0x94b3198>)>
<DelayedCall 0x94b1fa8 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('a.root-servers.net', <Deferred at 0x94b1f80>)>
<DelayedCall 0x94b3080 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('b.root-servers.net', <Deferred at 0x94b3058>)>
<DelayedCall 0x94b3238 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('e.root-servers.net', <Deferred at 0x94b3210>)>
<DelayedCall 0x94b3350 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('f.root-servers.net', <Deferred at 0x94b32d8>)>
<DelayedCall 0x94b3620 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('h.root-servers.net', <Deferred at 0x94b35a8>)>
<DelayedCall 0x94b3bc0 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('l.root-servers.net', <Deferred at 0x94b3b48>)>
<DelayedCall 0x94b3120 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('c.root-servers.net', <Deferred at 0x94b30f8>)>
<DelayedCall 0x94b34b8 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('g.root-servers.net', <Deferred at 0x94b3440>)>
<DelayedCall 0x94b3788 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('i.root-servers.net', <Deferred at 0x94b3710>)>
<DelayedCall 0x94b38f0 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('j.root-servers.net', <Deferred at 0x94b3878>)>
<DelayedCall 0x94b3a58 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('k.root-servers.net', <Deferred at 0x94b39e0>)>
<DelayedCall 0x94b3d28 [60.0s] called=0 cancelled=0 ThreadedResolver._cleanup('m.root-servers.net', <Deferred at 0x94b3cb0>)>
```

This suggests some kind of DNS rate-limiting or something.

<details><summary>Searchable metadata</summary>

```
trac-id__8663 8663
type__defect defect
reporter__glyph glyph
priority__normal normal
milestone__None None
branch__8663_namestaponwin 8663-namestaponwin
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__None None
time__1469327438408031 1469327438408031
changetime__1469444904946961 1469444904946961
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

It's always failed, but the Buildbot builders have just had warnonunclean to be off.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)|@hawkowl set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

I think that in this branch we should also update appveyor.yml to no longer ignore the errors ... so that the real status is reported for the commit

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

but maybe we should not cache openssh ... as it might be fast to install and in Travis-CI docs they say:

Large files that are quick to install but slow to download do not benefit from caching, as they take as long to download from the cache as from the original source.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban commented|
|-|-|

Please merge. All good. Thanks!

Maybe we can update appveyor.yml to report the real results ... but is optional

Many thanks for this great work!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|GitHub set status to `closed`|
|-|-|

In changeset 5b10ed051c6d0d04711b77179b65a8867797fe2e
```
#!CommitTicketReference repository="" revision="5b10ed051c6d0d04711b77179b65a8867797fe2e"
Merge pull request #406 from twisted/8663-namestaponwin

Author: hawkowl
Reviewers: adiroiban, rodrigc
Fixes: #8663
```
